fhir-server-spi/src/main/java/com/ibm/fhir/server/spi/operation/FHIRResourceHelpers.java

```diff
@@ -381,7 +381,25 @@ default Resource doVRead(String type, String id, String versionId) throws Except
      * @return a Bundle containing the history of the specified Resource
      * @throws Exception
      */
-    Bundle doHistory(MultivaluedMap<String, String> queryParameters, String requestUri) throws Exception;
+    default Bundle doHistory(MultivaluedMap<String, String> queryParameters, String requestUri) throws Exception {
+        return doHistory(queryParameters, requestUri, null);
+    }
+
+    /**
+     * Implement the system level history operation to obtain a list of changes to resources
+     * with an optional resourceType which supports for example [base]/Patient/_history
+     * requests to return the complete history of changes filtered to a specific resource type
+     *
+     * @param queryParameters
+     *            a Map containing the query parameters from the request URL
+     * @param requestUri
+     *            the request URI
+     * @param resourceType
+     *            optional resourceType to filter the history
```

Comment:
is this a comma-separated list if you're filtering on more than one?  if so, I think we should mention that here

Reply:
after reading the impl, it looks like param is only used for the resourceType path parameter and that the optional resourceTypes query parameter is passed separately (in the query params which are then baked into the context via FHIRPersistenceUtil.parseSystemHistoryParameters).